### ISSUE TITLE
Alignment using hillshading can give wrong results

### ISSUE THREAD

**Issue by oleg-alexandrov** [NeoGeographyToolkit/StereoPipeline#287]
The pc_align tool with the --initial-transform-from-hillshading option creates match points among hill-shaded images which are used to find an initial alignment. This function has outlier filtering, but it is not sufficiently robust. It should use RANSAC like we do in a lot of other places instead of doing a more naive filter of removing points that are not within the 75-th percentile with a factor. Normally this is good enough but can fail when there are just a handful of points.

**Comment by adehecq** [comment]
I thought the matching step in pc_align was running a RANSAC filtering. Isn't that the aim of the '--ransac-iterations 10000' argument in '--ipmatch-options'? I'm working with very difficult cases and pc_align has been really good at throwing away outliers so far, but with custom values for these parameters: --ipmatch-options "--inlier-threshold 10 --ransac-constraint similarity --ransac-iterations 100000"

**Comment by adehecq** [comment]
Yes, I had good results with a threshold of 10, but I try to have a reference DEM that has approximately the same resolution as my input DEM. 
Good to know that with the new RANSAC step, this threshold can be increased a bit.
